cloudbridge/cloud/providers/gce/services.py

```diff
@@ -435,12 +435,12 @@ def create(self, name, image, instance_type, zone=None,
                                         'name': 'External NAT'}]
                  }],
             }
-            if (security_groups is not None and
+            if (security_groups and
                 isinstance(security_groups, list) and
                 len(security_groups) > 0):
```

Comment:
I think that checking len(security_groups) isn't needed here.
If security_groups is an empty list then "if security_groups" returns False.